data/tools/pywmlx/state/wml_states.py

```diff
@@ -24,42 +24,6 @@ def run(self, xline, lineno, match):
 
 
 
-'''
```

Comment:
this is comment removal, it does not affect any code

Reply:
Have you read #8867?

Reply:
Yes, the fact that python validates comment syntax does not change the fact that class WmlDefineState did not exist before, and will not exist after.

Reply:
So, what change would you propose to this PR?